code/modules/surgery/implant.dm

```diff
@@ -181,7 +181,7 @@
 		for(var/datum/wound/wound in affected.wounds)
 			if(LAZYLEN(wound.embedded_objects))
 				loot |= wound.embedded_objects
-			find_prob += 50
+			find_prob +=85
```

Comment:
I'm fine with this, it's really annoying have to use the hemostate 4+ times to remove a single bullet because you failed a coin toss.